src/extension.ts

```diff
@@ -142,9 +142,11 @@ export async function activate(extensionContext: ExtensionContext): Promise<any>
         OdoImpl.Instance.loadWorkspaceComponents(event);
     });
 
-    OdoImpl.Instance.loadWorkspaceComponents(null);
+    await ComponentTypesView.instance.getAllComponents().then(() => {
```

Comment:
activate function should exit fast, there should be no waiting for long running calls like this. getAllComponents call is slow and that is the reason for test failure in CI.